### PR TITLE
[Dropdown,Search] Added Security Tab , evil example backend, hints

### PR DESCRIPTION
**Pull request by lubber-de**
## Description
This PR adds a security-tab to explain the `preserveHTML` setting and demonstrates possible XSS attacks in the dropdown & search module

## Screenshots
xss_proposaltext_v2
xss_hint
dropdown_xss_example
evilsearch
evildropdown

## Closes
https://github.com/Semantic-Org/Semantic-UI/issues/6570

## Relates to
https://github.com/fomantic/Fomantic-UI/pull/298

### CODE REVIEW

File: server/documents/modules/dropdown.html.eco [hunk 1]
```diff
@@ -3385,4 +3395,83 @@ themes      : ['Default', 'GitHub', 'Material']
...
+          url: '//api.github.com/search/repositories?q={query}'
+        }
+      });
+    </div>
+
+	<h2 class="ui dividing header">Evil API Example</h2>
+	<p>The following dropdown fields are bound to an evil API backend which tries to inject code</p>
+	<div class="ui message">
+		<ul class="ui list">
+			<li>Click into the dropdown field to trigger the api call</li>
```
Comment (hammy2899) [inline server/documents/modules/dropdown.html.eco]: "Click on the dropdown field"

File: server/documents/modules/search.html.eco [hunk 2]
```diff
@@ -944,4 +953,100 @@ type        : 'UI Module'
...
+    	},
+    	minCharacters : 3
+      });
+    </div>
+
+	<h2 class="ui dividing header">Evil API Example</h2>
+	<p>The following search fields are bound to an evil API backend which tries to inject code</p>
+	<div class="ui message">
+		<ul class="ui list">
+			<li>Enter at least 2 characters in the searchfield to trigger the api call</li>
```
Comment (hammy2899) [inline server/documents/modules/search.html.eco]: API

File: server/raw/evilapi.json [hunk 3]
```diff
@@ -0,0 +1,1000 @@
...
+    "name": "Wren",
+    "value": "wren"
+  }, {
+    "name": "Yak",
+    "value": "yak"
+  }, {
+    "name": "Zebra",
+    "value": "zebra"
+  }]
+}
```
Comment (hammy2899) [inline server/raw/evilapi.json]: Add new line at the end

**Comment by lubber-de** [comment]
Fixed. Thanks again for proof reading :slightly_smiling_face:

**Pull request merged**